source/val/validation_state.cpp

```diff
@@ -1319,5 +1319,183 @@ bool ValidationState_t::IsValidStorageClass(
   return true;
 }
 
+#define VUID_WRAP(vuid) "[" #vuid "] "
+
+std::string ValidationState_t::VkErrorID(uint32_t id,
+                                         const char* reference) const {
+  if (!spvIsVulkanEnv(context_->target_env)) {
+    return "";
+  }
+
+  // Currently no 2 VUID shared id so no need for reference
+  (void)reference;
```

Comment:
Nit: Add this comment at the start of the function and remove the parameter name.